### DISCUSSION TITLE
Pull request status freezes and does not react to new changes

### DISCUSSION THREAD

**Discussion by tabokie** [community/community#60103] [Pull Requests]
### Select Topic Area

Bug

### Body

The PR in question is https://github.com/tikv/rust-rocksdb/pull/754. It doesn't react to any new changes on the branch, and the status remains yellow when all checks have passed.

It seems concurrent "merge with master using GitHub UI" and "PR approval" triggered this PR to enter this state. I have to open another PR using a cloned branch to work around it.

Screenshot in case it recovers later:

Frozen PR:
![image](https://github.com/community/community/assets/27005812/17fdb58b-c015-4773-a71a-b63cbeeeb73c)

The branch:

![image](https://github.com/community/community/assets/27005812/228af9e4-6916-4e71-b9eb-ee8b14e55d7a)

**Comment by GhostOf0days** [comment]
Hello. There are currently ongoing issues with pull requests. Github is [investigating](https://www.githubstatus.com/incidents/pr0ptqcw6q8d).
